Lib/glyphsLib/interpolation.py

```diff
@@ -127,7 +127,7 @@ def get_axes(masters, instances):
                         userLoc = float(param.get('value', DEFAULT_LOC))
                         break
                 mapping.append((userLoc, interpolLoc))
-            mapping.sort()
+            mapping = list(sorted(set(mapping)))  # avoid duplicates
```

Comment:
no need to do `list(sorted(...))`, as `sorted` returns a new list in both python2 and 3.